src/PrestaShopBundle/Translation/PrestaShopTranslatorTrait.php

```diff
@@ -53,28 +53,24 @@ public function trans($id, array $parameters = [], $domain = null, $locale = nul
             $legacy = $parameters['legacy'];
             unset($parameters['legacy']);
         }
+        $emptyParams = empty($parameters);
+        $isSprintf = !$emptyParams && $this->isSprintfString($id);
```

Comment:
I think this can be simplified like this:
```suggestion
        $isSprintf = !empty($parameters) && $this->isSprintfString($id);
```